src/NuGet.Clients/PackageManagement.PowerShellCmdlets/Cmdlets/UpdatePackageCommand.cs

```diff
@@ -272,8 +272,9 @@ private void PerformPackageUpdatesOrReinstalls()
             foreach (var project in Projects)
             {
                 var installedPackages = await project.GetInstalledPackagesAsync(Token);
+
                 if (installedPackages.Select(installedPackage => installedPackage.PackageIdentity.Id)
-                                     .Any(installedPackageId => installedPackageId.Equals(packageId, StringComparison.OrdinalIgnoreCase)))
+                    .Any(installedPackageId => installedPackageId.Equals(packageId, StringComparison.OrdinalIgnoreCase)))
```

Comment:
Would it be better if this was simplified to - 

`if (installedPackages.Any(installedPackage => installedPackage.PackageIdentity.Id.Equals(packageId, StringComparison.OrdinalIgnoreCase)))`